docs/user-guide/cai.rst

```diff
@@ -258,7 +255,7 @@ This command will create the ``rl-cai`` dataset files in the defined output fold
 
 
 Step 5: Train the RM
```

Comment:
Line 260/257 revise. define acronym which is useful for user searches

Step 5: Train the Reward Model (RM)

Reply:
https://github.com/NVIDIA/NeMo-Aligner/pull/411/commits/9432333a44ad01d9534ed89263a37000b2aaca11